app/src/main/AndroidManifest.xml

```diff
@@ -9,18 +9,35 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".activity.ChatRoomActivity"></activity>
-        <activity android:name=".activity.ViewUserProfileActivity"></activity>
-        <activity android:name=".activity.BlockedUsersActivity" />
-        <activity android:name=".activity.EditMyProfileActivity" />
-        <activity android:name=".activity.ViewMyProfileActivity" />
-        <activity android:name=".activity.myBook.AddNewBookActivity" />
-        <activity android:name=".activity.myBook.EditBookActivity" />
-        <activity android:name=".activity.SignUpActivity"/>
+        <activity
+            android:name=".activity.ChatRoomActivity"
+            android:screenOrientation="portrait"></activity>
+        <activity
+            android:name=".activity.ViewUserProfileActivity"
+            android:screenOrientation="portrait"></activity>
+        <activity
+            android:name=".activity.BlockedUsersActivity"
+            android:screenOrientation="portrait" />
+        <activity
+            android:name=".activity.EditMyProfileActivity"
+            android:screenOrientation="portrait" />
+        <activity
+            android:name=".activity.ViewMyProfileActivity"
+            android:screenOrientation="portrait" />
+        <activity
+            android:name=".activity.myBook.AddNewBookActivity"
+            android:screenOrientation="portrait" />
+        <activity
+            android:name=".activity.myBook.EditBookActivity"
+            android:screenOrientation="portrait" />
+        <activity
+            android:name=".activity.SignUpActivity"
+            android:screenOrientation="portrait" />
         <activity
             android:name=".activity.MainActivity"
-            android:windowSoftInputMode="adjustPan"
-            android:label="@string/app_name">
+            android:label="@string/app_name"
+            android:screenOrientation="portrait"
```

Comment:
Sadly there is not a cleaner way that setting each activity to disable screen rotate